### PR TITLE
Add no current user tests

### PR DESCRIPTION
**Pull request by tsteer**

### CODE REVIEW

File: app/views/articles/index.html.erb [hunk 1]
```diff
@@ -2,9 +2,12 @@
   <p>There are no articles</p>
 <% else %>
   <% @articles.each do |f| %>
-    <%= f.title %>
-    <%= f.subheading %>
-    <%= f.author %>
-    <%= f.article_text %>
+    <div class="card mb-3">
+      <div class="card-block px-3 py-2">
+        <h1 class="card-title py-2"><%= link_to f.title, article_path(f) %></h1>
```
Comment (craigjbass) [inline app/views/articles/index.html.erb]: why h1 -> h3 -> h6 ?

File: app/views/forum_categories/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
   <p>There are no categories</p>
 <% else %>
-  <% @forum_categories.each do |f| %>
-    <%= link_to f.title, forum_category_path(f) %>
-  <% end %>
+  <div class="card mb-3">
+    <div class="card-block px-3 py-2">
+      <% @forum_categories.each do |f| %>
+        <%= link_to f.title, forum_category_path(f), class: 'card-title' %>
+        &emsp;
```
Comment (craigjbass) [inline app/views/forum_categories/index.html.erb]: Feels like could be a job for CSS?

File: spec/features/users/without_a_user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,267 @@
+require 'rails_helper'
+
+RSpec.feature 'accessibility without a user' do
+
```
Comment (craigjbass) [inline spec/features/users/without_a_user_spec.rb]: blank line

File: app/views/forum_categories/index.html.erb (hunk 2 above)
Comment (tsteer) [reply app/views/forum_categories/index.html.erb]: True!

**Pull request merged**